Example/Player Example/ViewController.swift

```diff
@@ -78,17 +78,17 @@ extension ViewController: DMPlayerViewControllerDelegate {
   fileprivate func toggleFullScreen() {
     isPlayerFullscreen = !isPlayerFullscreen
     updateDeviceOrientation()
-    updatePlayerSize()
+    updatePlayerSize(self.view.frame.size)
   }
   
   private func updateDeviceOrientation() {
     let orientation: UIDeviceOrientation = isPlayerFullscreen ? .landscapeLeft : .portrait
     UIDevice.current.setValue(orientation.rawValue, forKey: #keyPath(UIDevice.orientation))
   }
   
-  fileprivate func updatePlayerSize() {
+  fileprivate func updatePlayerSize(_ size : CGSize) {
```

Comment:
Come to think of it, the method name is a bit of a misnomer. Let's do this instead: `func updatePlayer(height: CGFloat)` and swiftify it in the process too.